src/components/property-info/property-info.ts

```diff
@@ -0,0 +1,19 @@
+import { colors } from '../../styles/colors';
+import { typography } from '../../styles/typography/typography';
+
+export const propertyStyles = {
+  display: 'flex',
+  backgroundColor: colors.NeutralXlight,
+  color: colors.NeutralXdark,
+  fontSize: typography.size.medium,
+  border: `1px solid ${colors.NeutralXlight}`,
+  alignItems: 'center',
+  justifyContent: 'center',
+  borderRadius: typography.border.radiusLarge,
+  width: '100px',
+  height: '95px',
```

Comment:
É legal evitar de fazer componentes com altura e largura fixa sempre que possível. Vou dar um exemplo de porque essa pode ser uma má prática: aqui se o título for grande, uma largura de `100px` pode acabar deixando o texto cortado, certo?

Um outro exemplo não muito a ver com esse: a altura também pode ter esse problema em componentes que o texto quebra linha, por exemplo.